### ISSUE TITLE
Ignore dist directory and Thumbs.db files

### ISSUE THREAD

**Issue by eduardo-matos** [sahat/requirejs-library#5]
It doesn't make sense to add the `dist/` directory to version control. For Windows users it doesn't make sense to add `Thumbs.db` files to version control as well.

**Comment by sahat** [comment]
I included `dist/` so people could download the repository and run demo right away without too much hassle. I could add `dist/` to _gitignore_, and move `mylib.min.js` to demo but then I either need to create a separate Grunt task to perform "copy" or else `mylib.min.js` in _demo_ folder will no longer match what you have in `dist/` after running `grunt` command.

Being this an example repository I think it's more helpful to include `dist/` folder than being pedantic about it.

But the second point is valid and I went ahead and added Thumbs.db to gitignore.
